components/cluster/command/check.go

```diff
@@ -261,6 +261,24 @@ func checkSystemInfo(s *cliutil.SSHConnectionProps, topo *spec.Specification, op
 					task.CheckTypePackage,
 					topo,
 					opt.opr,
+				).
+				// check for THP
+				Shell(
+					inst.GetHost(),
+					"cat /sys/kernel/mm/transparent_hugepage/enabled",
+					false,
+				).
+				Shell(
+					inst.GetHost(),
+					"cat /sys/kernel/mm/transparent_hugepage/defrag",
```

Comment:
1. Maybe you can merge the two command into one:

```bash
cat /sys/kernel/mm/transparent_hugepage/{enabled,defrag}
```

2. IMO, maybe you can implement the THP Check like `CheckSELinux()`